docs/src/pages/[platform]/theming/theme-provider/examples/BasicExample.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { Card, Text } from '@aws-amplify/ui-react';
+
+export const BasicExample = () => (
+  // Example using component variations
```

Comment:
Explaining variations seems out of place here. Could we maybe show `useTheme` hook as a second way to access theme information. One example I've used is 
```
border={`1px solid ${tokens.colors.brand[80]}`}
```